src/AssemblySharedInfoGenerator/AssemblyInfoGenerator.csproj

```diff
@@ -59,8 +59,18 @@
     <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <!--Checking if the Text Template transform is installed with VS IDE so it is only for Dev machines-->  
+  <Import Condition="Exists('$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v16.0\TextTemplating\Microsoft.TextTemplating.targets')" Project="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v16.0\TextTemplating\Microsoft.TextTemplating.targets" />
```

Comment:
so @alfredo-pozo - does this target build all .tt files it finds in the project? In the repo? What does it do exactly?

Reply:
What it does on that line is that it looks for the targets file that triggers the execution of the transformation of the templates related to that project on the location that a computer with VS IDE installs the Text Templating Transform feauture, otherwise, it will look for the location that is on the extern folder, where the CI pipeline installs the TextTemplating nuget  we publish on ADSK artifactory.

Reply:
gotcha:
for anyone who looks at this later:
https://github.com/github/VisualStudio/blob/master/lib/Microsoft.TextTemplating.targets#L296

the way this is setup, because @alfredo-pozo set `TransformOnBuild` to true -  it will call its own `TransformAll` target which transforms all the `.tt` files in the current project on each build.

Reply:
@mjkkirschner: https://git.autodesk.com/Dynamo/LibG/pull/901